src/client/pages/OfferNew/Checkout/InsuranceSummaryDetails.tsx

```diff
@@ -200,6 +203,29 @@ const getQuoteDetails = (
   return detailsGroups
 }
 
+export const getAddress = (quoteDetails: QuoteDetails) => {
+  if (!quoteDetailsHasAddress(quoteDetails)) {
+    return null
+  }
+
+  const { street } = quoteDetails
+
+  if ('floor' in quoteDetails && 'apartment' in quoteDetails) {
+    const { floor, apartment } = quoteDetails
+
+    if (floor || apartment) {
+      const formattedFloor = floor ? `${floor}.` : ''
+      const apartmentString = apartment ? ` ${apartment}` : ''
+
+      return `${street}, ${formattedFloor}${apartmentString}`
+    }
+
+    return street
+  }
```

Comment:
Wouldn't this be the same?

```suggestion
  const { floor, apartment } = quoteDetails 
  if (floor || apartment) {
    const formattedFloor = floor ? `${floor}.` : ''
    const apartmentString = apartment ? ` ${apartment}` : ''

    return `${street}, ${formattedFloor}${apartmentString}`
  }
```

Reply:
I would also add some comment here that this is Danish-specific logic since it lives outside the localisation module.

Reply:
Wait, I feel blind - you just removed space, right?

Reply:
You can't remove the `in` check I think since those properties don't exist on all type of `quoteDetails` but only the Danish version.

Reply:
And I feel like this is an edge case if it should live in the localisation module or not - maybe we want to display apartment number for other markets as well?

Reply:
Right, I just mean that other markets most likely format their addresses differently ;)

And regarding the `in` checks; won't the values just be `undefined` for non-Danish quotes?

Reply:
No, TypeScripts yells if you remove the `in` checks since those properties don't even exist on e.g. `SwedishApartmentQuoteDetails`.

But yes, you are of course correct that other markets might format their addresses differently. Maybe somewhat of a pre-optimisation to prepare for that now though? Or maybe not 🤔